htdocs/takepos/invoice.php

```diff
@@ -818,7 +818,10 @@ function DolibarrTakeposPrinting(id) {
 			$adh->ref = $adh->getFullName($langs);
 			$s .= $adh->getFullName($langs);
 			$s .= ' - '.$adh->type;
-			if ($adh->datefin)
+      		if (empty($adh->statut) && $adh->datefin) {
+				$s .= '<br>'.$langs->trans("MemberStatusResiliated");
```

Comment:
To try to save spaces on the top bar, what do you think of striking the name of the member when he is terminated ?

Reply:
@eldy i changed it for strike all (name, type and datefin for more visibility)